tests/fgcmcalTestBase.py

```diff
@@ -551,6 +549,94 @@ def _testFgcmCalibrateTract(self, visits, tract,
         self.assertFalse(butler.datasetExists('fgcmStarIndices'))
         self.assertFalse(butler.datasetExists('fgcmReferenceStars'))
 
+    def _compareBuildStars(self, butler1, butler2):
+        """
+        Compare the full set of BuildStars outputs with files from two
+        repos.
+
+        Parameters
+        ----------
+        butler1 : `lsst.daf.persistence.Butler`
+        butler2 : `lsst.daf.persistence.Butler`
+
+        Raises
+        ------
+        Exceptions on test failures.
+        """
+        import numpy.testing as testing
+        import esutil
+
+        # Check the visit catalogs are identical
+        visitCat1 = butler1.get('fgcmVisitCatalog').asAstropy()
+        visitCat2 = butler2.get('fgcmVisitCatalog').asAstropy()
+
+        for col in visitCat1.columns:
+            if isinstance(visitCat1[col][0], str):
+                testing.assert_array_equal(visitCat1[col], visitCat2[col])
+            else:
+                testing.assert_array_almost_equal(visitCat1[col], visitCat2[col])
+
+        # Check that the observation catalogs have the same length
```

Comment:
Why just length? If that's all that's testable, please spell it out in a comment.